src/BlazorBoilerplate.Server/Controllers/EmailController.cs

```diff
@@ -16,81 +12,24 @@ namespace BlazorBoilerplate.Server.Controllers
     [ApiController]    
     public class EmailController : ControllerBase
     {
-        // Logger instance
-        ILogger<EmailController> _logger;
-        IEmailService _emailService;
+        private readonly IEmailManager _emailManager;
 
-        public EmailController(IEmailService emailService, ILogger<EmailController> logger)
+        public EmailController(IEmailManager emailManager)
         {
-            _logger = logger;
-            _emailService = emailService;
+            _emailManager = emailManager;
         }
 
         [HttpPost("Send")]
         [ProducesResponseType(204)]
         [ProducesResponseType(400)]
-        public async Task<ApiResponse> Send(EmailDto parameters)
-        {
-            if (!ModelState.IsValid)
-            {
-                return new ApiResponse(400, "User Model is Invalid");
-            }
-
-            var email = new EmailMessageDto();
-
-            email.ToAddresses.Add(new EmailAddressDto(parameters.ToName, parameters.ToAddress));
-
-            //This forces all emails from the API to use the Test template to prevent spam
-            parameters.TemplateName = "Test";
-
-            //Send a Template email or a custom one since it is hardcoded to Test template it will not do a custom email.
-            if (!string.IsNullOrEmpty(parameters.TemplateName))
-            {
-                switch (parameters.TemplateName)
-                {
-                    case "Test":
-                        email.BuildTestEmail(); //example of email Template usage
-                        break;
-                    default:
-                        break;
-                }
-            }
-            else
-            {
-                email.Subject = parameters.Subject;
-                email.Body = parameters.Body;
-            }
-
-            //Add a new From Address if you so choose, default is set in appsettings.json
-            //email.FromAddresses.Add(new EmailAddress("New From Name", "email@domain.com"));
-            _logger.LogInformation("Test Email: {0}", email.Subject);
-            try
-            {
-                await _emailService.SendEmailAsync(email);
-                return new ApiResponse(200, "Email Successfuly Sent");
-            }
-            catch (Exception ex)
-            {
-                return new ApiResponse(500, ex.Message);
-            }
-
-
-        }
+        public async Task<ApiResponse> Send(EmailDto parameters) 
+            => ModelState.IsValid ?
+                await _emailManager.Send(parameters) :
+                new ApiResponse(400, "User Model is Invalid");
 
         [HttpGet("Receive")]
         [Authorize]
-        public async Task<ApiResponse> Receive()
-        {
-            //Check email from default account defined in appsettings.json
```

Comment:
@DanielBunting  I do like the leaner code with lambda functions but as boilerplate I feel that we need to keep the comments in the code for newbies. You wouldn't believe how many questions I get between BB and MatBlazor.